server/app.js

```diff
@@ -312,6 +340,35 @@ app.get('/user/:username', async (req, res, next)=>{
 	return next();
 });
 
+//Rename Brews
+app.put('/api/user/rename', async (req, res) => {
+    const { username, newUsername } = req.body;
```

Comment:
Might be good to check if `req.account` exists here to make sure only logged-in users are changing names. Not sure if we need to do anything special to get the logged in user if the request comes from naturalcrit but they should share the same cookie I think, so the JWT handler above should just work.

Reply:
I think it's worth it to keep it inline with the other APIs we have for user account features. Should just be a 1-line addition here I believe.

Reply:
will fix this in the next PR, cannot test if the req.account is getting through in a deployment